jarviscli/plugins/htmltopdf_url.py

```diff
@@ -0,0 +1,30 @@
+import pdfkit
+from plugin import plugin, require, LINUX
+
+
+@require(platform=LINUX, native=["wkhtmltopdf"], network=True)
+@plugin("htmltopdf url")
+class htmltopdf_url:
+    
+    """transform your url page into a pdf file in the Jarvis source directory. type your url as the following:
+    'htmltopdf google.com'
+    The output file will be the following:
+    'google.com.pdf'"""
+
+    def __call__(self, jarvis, s):
+    	
+    	#We have to add the '.' back beacause the Jarvis API removes it
```

Comment:
Year, I'll implement a way that Jarvis API e.g. does not remove '.' of something that looks like an URL ;).